packages/commonwealth/client/scripts/views/pages/manage_community/chain_metadata_rows.tsx

```diff
@@ -155,13 +154,13 @@ const DiscordForumConnections = ({
   );
 };
 
-export const ChainMetadataRows = ({
-  chain,
+export const CommunityMetadataRows = ({
```

Comment:
if you change component name, also please do change tsx/scss file name

Reply:
Great catch.